src/installing-workflow/experimental-native-ingress.md

```diff
@@ -0,0 +1,80 @@
+# Experimental Native Ingress
```

Comment:
So I already considered hooking into the existing page, I think it makes sense to have a clear change in flow for an entire section, than to try to add a bunch of conditional logic to an existing flow. I visualize the flows like this:

### Separate and clear
```
              ----
----- <           > ------
              ----
```

### Together and conditional

```
---- <>-<>-<>-<>
```